darts/models/forecasting_model.py

```diff
@@ -140,25 +140,30 @@ def _backtest_model_specific_sanity_checks(self, *args: Any, **kwargs: Any) -> N
         pass
 
     @_with_sanity_checks("_backtest_sanity_checks", "_backtest_model_specific_sanity_checks")
-    def backtest(self,
-                 training_series: TimeSeries,
-                 target_series: Optional[TimeSeries] = None,
-                 start: Union[pd.Timestamp, float, int] = 0.7,
-                 forecast_horizon: int = 1,
-                 stride: int = 1,
-                 retrain: bool = True,
-                 trim_to_series: bool = True,
-                 verbose: bool = False,
-                 use_full_output_length: Optional[bool] = None) -> TimeSeries:
-        """ Retrain and forecast values pointwise with an expanding training window over `series`.
+    def historical_forecasts(self,
+                             training_series: TimeSeries,
+                             target_series: Optional[TimeSeries] = None,
+                             start: Union[pd.Timestamp, float, int] = 0.7,
+                             forecast_horizon: int = 1,
+                             stride: int = 1,
+                             retrain: bool = True,
+                             overlapp_series_end: bool = False,
+                             last_points_only: bool = False,
```

Comment:
I would consider setting the default to `True`, because calling with `False` would mostly be for the backtest() function, and for users who know what they are doing. For many users it would be nice to just get the single `TimeSeries` of historical last-point forecasts I think.